c10/core/QEngine.h

```diff
@@ -17,14 +17,15 @@ enum class QEngine : uint8_t {
   QNNPACK = 2,
   ONEDNN = 3,
   X86 = 4,
+  Arm = 5,
```

Comment:
are these needed for pt2e quant stack? I feel these are only needed for the older fx stack

Reply:
Yes, I think it's only needed for old stacks.

Reply:
Same for qconfig.